### PR TITLE
ISIS Ephemeris Times

### PR DESCRIPTION
**Pull request by acpaquette**
Updates ALE to produce the same set of ephemeris times that ISIS produces before any reduction on the pointing/position data.

### CODE REVIEW

File: ale/base/type_sensor.py [hunk 1]
```diff
@@ -237,9 +241,12 @@ def ephemeris_time(self):
...
         """
-        # 0.25 is the delta used by minirf, used as a default.
-        num_states = int((self.ephemeris_stop_time - self.ephemeris_start_time)/0.25) + 1
-        return np.linspace(self.ephemeris_start_time,  self.ephemeris_stop_time, num_states)
+        if not hasattr(self, "_ephemeris_time"):
+          cache_size = self.image_lines + 1
+          cache_slope = (self.ephemeris_stop_time - self.ephemeris_start_time) / (cache_size - 1)
+          self._ephemeris_time = np.asarray([self.ephemeris_start_time +  (i * cache_slope) for i in range(cache_size)])
+        return self._ephemeris_time
+        # return np.linspace(self.ephemeris_start_time,  self.ephemeris_stop_time, num_states)
```
Comment (jessemapel) [inline ale/base/type_sensor.py:249]: Remove this old commented line

File: tests/pytests/test_cassini_drivers.py [hunk 2]
```diff
@@ -28,6 +28,7 @@ def test_load_pds(test_kernels):
     isd_obj = json.loads(isd_str)
     print(json.dumps(isd_obj, indent=2))
     x = compare_dicts(isd_obj, compare_dict)
+    print(x)
```
Comment (jessemapel) [inline tests/pytests/test_cassini_drivers.py:31]: remove debug prints in this driver

File: tests/pytests/test_data_naif.py [hunk 3]
```diff
@@ -105,6 +105,7 @@ def test_sensor_orientation(self):
         self.driver.ephemeris_time = [297088762.61698407]
         self.driver._props = {}
         orientation = self.driver.sensor_orientation
+        print("{:.16f}, {:.16f}, {:.16f}, {:.16f}".format(*orientation[0]))
```
Comment (jessemapel) [inline tests/pytests/test_data_naif.py:108]: does the numpy assert not give enough detail for this?

**Review by jessemapel — COMMENTED** [review]
Changes look good. I'm concerned about the size of the test data, particularly the marci one. Can we potentially trim it to every 100th or something position/pointing before we compare it?

File: tests/pytests/test_data_naif.py (hunk 3 above)
Comment (acpaquette) [reply tests/pytests/test_data_naif.py:108]: Revmoved

**Review by jessemapel — APPROVED** [review]

**Pull request merged**